File: Python Folder/PRAK205-2210817310011-M.Daffa_Az-Zikra.py
from math import sqrt
a,b = input().split()
a = int(a)
b = int(b)
c = sqrt((b*b)-(a*a))
k = a+b+c
l = (a*c)/2
c = int(c)
k = int(k)
l = int(l)
print('Alas =',c,'cm\nTinggi =',a,'cm')
print('Keliling =',k,'cm\nLuas =',l,'cm^2')

File: C Folder/PRAK205-2210817310011-M.Daffa_Az-Zikra.c
#include<stdio.h>
#include<math.h>
int main(){
    double a,b,c,k,l;
    scanf("%lf %lf",&a,&b);
    c = sqrt((b*b)-(a*a));
    k = a+b+c;
    l = (a*c)/2;
    printf("Alas = %.lf cm\nTinggi = %.lf cm\n",c,a);
    printf("Keliling = %.lf cm\nLuas = %.lf cm^2",k,l);
    return 0;
}